src/graph.h

```diff
@@ -26,14 +26,14 @@ class Graph : public torch::CustomClassHolder {
   void SetCSC(std::shared_ptr<CSC> csc);
   void SetCSR(std::shared_ptr<CSR> csr);
   void SetCOO(std::shared_ptr<COO> coo);
-  void SetData(torch::Tensor data);
+  void SetData(torch::Tensor data, std::string order = "default");
```

Comment:
is direction better than order?

Reply:
what's default value?

Reply:
`default` is the original data sequence without explicit edge id indexing of typical format. It is used where user just want to get the data without taking care of the data sequence order.

Reply:
Same as DGL, in every matrix there is a implicit default data order with the `e_id` of corresponding format being `torch::nullopt`. For example, if the default order is `coo`, `coo_->e_ids` will be `torch::nullopt`, if we want to get the data sequence order of csc format, there will be a explicit edge id indexing.

Reply:
okkk